extensions/adobe/experience/campaign/experienceevent.schema.json

```diff
@@ -255,8 +270,17 @@
               "type": "string",
               "description":
                 "Business qualifier that identifies the event sent by the data source. It informs on the business reason for sending the event. It is unique per organization. This is used by Campaign orchestration to identify the event without inspecting its payload to determine which action should be triggered when the event is received. The value of this field is a contract between Campaign orchestration and the data source."
-            }
+            },
+            "orchestrationDetails": {
+              "title": "Business Reason Details",
+              "$ref": "https://ns.adobe.com/experience/campaign/orchestration/experienceevent#/definitions/journeyContext",
+              "description": "Set of attributes that are associated with every business reason."
           }
+        },
+        "https://ns.adobe.com/experience/campaign/containerID": {
```

Comment:
Here, container acts as a kind of segregator to make separation between campaign experience events from each other. It can be used to represent tenantID in multi tenant applications which clearly distinguishes one tenant data from other tenant.